Gemfile.lock

```diff
@@ -2,14 +2,15 @@ GEM
   remote: https://rubygems.org/
```

Comment:
A fine question.  Including the lock file is a best practice.  Bundler makes it really easy to update versions.  Including the lock file makes it easy to freeze the versions for testing / deployment.